Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.cs

```diff
@@ -234,6 +234,11 @@ private void FocusContainerFromIndex(int index)
         }
 
         /// <summary>
+        /// Occurs when the selected item has changed
+        /// </summary>
+        public event EventHandler<object> SelectionChanged = null;
```

Comment:
Would it make sense to add a new type for the eventargs that will contain the index and the selectedItem?